osf_tests/test_user.py

```diff
@@ -492,6 +492,33 @@ def test_display_full_name_unregistered(self):
         u.reload()
         assert u.display_full_name(node=project) == name
 
+    def test_repeat_add_same_unreg_user_with_diff_name(self):
+        u = UnregUserFactory()
```

Comment:
Minor, not blocking suggestion; We're trying to be more consistent and descriptive in test variable names, putting them in the format <what it is>_<descriptive features> (e.g., `user_one`, `user_two`, `user_admin`, `user_read` etc)

For this one, `user_unregistered` would be more ideal.

Look over some of Rheisen's pytest conversions for more examples